components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -105,6 +104,8 @@ - (void)viewDidLoad {
   contentViewLabel.text = @"Content View";
   [contentViewLabel sizeToFit];
   contentViewLabel.center = contentView.center;
+  self.contentViewLabel = contentViewLabel;
```

Comment:
Given the complexity of your views, I think it's better to make a custom UIView and do the layout there. Then you can simplify the example VC to focus on how to use a Banner.

Reply:
Sure. I created an issue https://github.com/material-components/material-components-ios/issues/6842, and will do it in a separate PR.